dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp

```diff
@@ -215,6 +271,21 @@ PrehandleResult KVStore::preHandleSnapshotToFiles(
     return result;
 }
 
+size_t KVStore::getMaxParallelPrehandleSize() const
+{
+    const auto & proxy_config = getProxyConfigSummay();
+    size_t total_concurrency = 0;
+    if (proxy_config.valid)
+    {
+        total_concurrency = proxy_config.snap_handle_pool_size;
```

Comment:
It is in https://github.com/pingcap/tidb-engine-ext/blob/521fd9dbc55e58646045d88f91c3c35db50b5981/proxy_components/proxy_server/src/config.rs#L50

The idea here is in raftstore-v2 scene, many table has only one snapshots, so if there the only region of this table is serving, TiFlash can't actually serve anything.

However, I think I can adopt the previous strategy, since it is less aggresive.